### PR TITLE
Error alerting

### PR DESCRIPTION
**Pull request by hanars**
Since we now alert on all logged errors, this cleans up some error handling

### CODE REVIEW

File: seqr/utils/elasticsearch/es_search.py [hunk 1]
```diff
@@ -691,7 +695,8 @@ def _is_matched_sample(family_guid, sample):
 
     def _parse_compound_het_response(self, response):
         if len(response.aggregations.genes.buckets) > MAX_COMPOUND_HET_GENES:
-            raise Exception('This search returned too many compound heterozygous variants. Please add stricter filters')
+            from seqr.utils.elasticsearch.utils import InvalidSearchException
```
Comment (hanars) [inline seqr/utils/elasticsearch/es_search.py:698]: this file can't be imported at the top of the file due to circular import issues

**Comment by hanars** [comment]
codacy (errors are okay to ignore): https://app.codacy.com/gh/macarthur-lab/seqr/pullRequest?prid=6517032

**Comment by hanars** [comment]
passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/744064817

File: seqr/views/utils/pedigree_image_utils_tests.py [hunk 2]
```diff
@@ -69,13 +83,51 @@ def _mock_paint(command):
...
         ])
+        mock_logger.warning.assert_not_called()
+        mock_logger.error.assert_called_with('Generated pedigree image for family 1 with exit status 1: Error!')
 
         # Do not generate for families with one individual
         mock_tempfile_file.write.reset_mock()
-        mock_os_system.reset_mock()
+        mock_run.reset_mock()
+        mock_logger.warning.reset_mock()
+        mock_logger.error.reset_mock()
```
Comment (ShifaSZ) [inline seqr/views/utils/pedigree_image_utils_tests.py:93]: These two lines can be `mock_logger.reset_mock()`.

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: seqr/utils/elasticsearch/es_search.py (hunk 1 above)
Comment (hanars) [reply seqr/utils/elasticsearch/es_search.py:698]: Nope, it just makes a circular dependency elsewhere :(

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**